cmd/standalone/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM golang:1.7
+FROM scratch
 # expose agent UDP ports and query port
 EXPOSE 5775/udp 6831/udp 6832/udp 16686
```

Comment:
hmmm, good catch. afaik hotrod doesn't use remote sampler in the client so it hasn't been an issue but I'll add it before I land this.